frontend/src/common/commonStyle.css

```diff
@@ -0,0 +1,43 @@
+    .gridItem{
```

Comment:
There's no reason to make a separate file and include it multiple times throughout the application.  Just put this in the index.scss and it will be available throughout.  If you need to override you can override them in the component and with specificity.

Reply:
Yes, I didn't think at first that putting in the index.css will available throughout. It works! Perfect!